FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/covg_single_priority_no_timeslice_utest.c

```diff
@@ -415,3 +415,109 @@ void test_task_step_tick_xNextTaskUnblockTime_not_equal( void )
     xTickCount = 0U;
     vTaskStepTick((TickType_t)10U);
 }
+
+/**
+ * @brief xTaskResumeFromISR - resume higher priority suspended task
+ *
+ * This test resume a higher priority task from ISR when scheduler suspended. The
+ * return value of xTaskResumeFromISR indicates yield required for the core calling
+ * this API.
+ *
+ * <b>Coverage</b>
+ * @code{c}
+ * prvYieldForTask( pxTCB );
```

Comment:
Should this be xTaskResumeFromISR() ?

Reply:
This is the code snippet in xTaskResumeFromISR().
I will add more code snippet to make it more clear to the readers.
https://github.com/chinglee-iot/FreeRTOS-Kernel/blob/b266846c84a18491a3f186d76a71eacacfe567d8/tasks.c#L3014-L3023